net-libs/mbedtls/files/mbedtls-3.6.2-add-version-suffix-for-all-installable-targets.patch

```diff
@@ -0,0 +1,86 @@
+From 9ebd03bfd9f3ae7f431f868a2246c4a1d71f3f07 Mon Sep 17 00:00:00 2001
```

Comment:
Please (always) include a link at the top of patches to any relevant upstream PRs/bugs or Gentoo bugs.

Reply:
This doesn't seem done?